CondTools/Ecal/python/EcalSCDynamicDPhiParametersMaker_cfg.py

```diff
@@ -0,0 +1,30 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
I think this file might belong better in /test/, it doesn't seem like something you would want to import from other configs.